include/eve/algo/traits.hpp

```diff
@@ -106,12 +106,13 @@ template <typename Traits, typename T>
     rbr::get_type_t<Traits, force_cardinal_key,
     eve::expected_cardinal_t<iteration_type_t<Traits, T>>>;
 
-  template <typename User, typename Default>
-  constexpr auto default_to(traits<User> const& user, traits<Default> const& defaults)
+
+  inline constexpr auto default_to =
```

Comment:
why a lambda instead of a function here ?

Reply:
To be able to pass into kumi::fold_right.

I was thinking to do a struct but seemed like lambda is enough.

I want to clean up all functions at some point

Reply:
OK